glasskube/trieve/cronjobs/clickhouse-collapse.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: collapsing-queries-cron
```

Comment:
most of the resources start with "trieve-". It might come in handy to also have these jobs follow the same naming convetions